.drone.yml

```diff
@@ -167,6 +179,10 @@ services:
     ATHENS_STORAGE_GCP_BUCKET: "athens_ci_build"
     ATHENS_STORAGE_GCP_JSON_KEY:
       from_secret: GCS_SERVICE_ACCOUNT
+volumes:
+- name: RedisConf
+  host:
```

Comment:
`host` is disallowed in Drone because it's a security risk to expose the underlying machine: https://docs.drone.io/pipeline/docker/syntax/volumes/

You'll need to use a temporary volume if you can https://docs.drone.io/pipeline/docker/syntax/volumes/temporary/

Reply:
Not sure how that will help me mount the `redis.conf` from the `./test` directory of the `repo`.

Reply:
@mplatcher try having no volumes at all, and just point the command to `test/redis.conf` as the working directory should be the same. 

If Drone doesn't give the volume to a `service` we can try to have it as a detached step: https://docs.drone.io/pipeline/docker/syntax/services/

Finally, if you'd like to disable test so we can merge feel free to do so. As a last resort, we can publish our own image on docker/gomods that already has the config file :)

Reply:
@marwan-at-work thanks so much didn't know they mount the root volume of the project to each service. that was super easy once I knew that.